### ISSUE TITLE
How to get relative paths in File panel

### ISSUE THREAD

**Issue by bartgol** [jesseduffield/lazygit#1573]
**Is your feature request related to a problem? Please describe.**
Context: I work on a repo with many nested folders, but only work in one of those. With vanilla `git status`, all the modified files are reported relative to  the repo root dir. I can change this with `git config status.relativePaths true`. This option is particularly handy for repos with long folder names. E.g., if you work only in the `$repo_root/long_string/another_long_string/maybe_a_third_one` folder.

**Describe the solution you'd like**
I am not sure what command lazygit runs to get the list of unstaged files in the `Files` panel. I suspect it's not `git status`, or else it should honor my global git config (right) for the output of status. But in general, I might want relative paths for lazygit even if I don't have them in my global/repo git config, since lazygit has panels, and the horiz space is limited, so shorter file names can be useful. To this end, it would be nice if lazygit had a config option to allow paths in the File panel that are relative to the current folder. This option could be under `gui` (something like `showFilesRelativePath`) or under `git`, but I don't know exactly what git command is run to retrieve that list (as I said, I suspected it was `git status`, but then it should have honored my configs, so maybe it's `git ls-files`)?

**Describe alternatives you've considered**
I don't have a clear idea of how to achieve this result. I would have to go check what git command lazygit uses to retrieve the diffing files, but that's intrusive, since it requires knowledge of internal lazygit impl details. It would be best for lazygit to offer a config option in its config.yml file.

**Additional context**
See screenshot for an example of this issue. I work 99.9% inside the `components/scream/src` folder of the repo; being able to remove that common prefix would allow to see the filenames more easily.
![Screenshot at 2021-11-11 12-56-56](https://user-images.githubusercontent.com/3226046/141360891-42b6203e-2947-497b-a67e-49b236959113.png)

**Comment by mjarkk** [comment]
Do you know about <kbd>`</kbd> it toggles the tree view and should make inspecting nested file changes easier

**Comment by bartgol** [comment]
Yes, the tree view is the default...but I deactivated it in my config.yml file, since I didn't really like it that much (it adds extra lines in a small window where space is already limited).

Note: I am not sure how easy/hard it would be to change the files list (in non-tree mode) to be relative to the folder where lazygit was invoked from. My guts tell me "not too much", assuming lazygit takes the output of `git status` or `git ls-files -m` and reformats it a bit. But I reckon I might be in the minority here, with most people fine with either tree mode or normal mode with full path...
